docs/general/queries.md

```diff
@@ -44,3 +44,57 @@ schema = strawberry.Schema(query=Query)
 ```
 
 So now, when requesting the name field, the `get_name` function will be called.
+
+Alternatively a field can be declared using a decorator:
+
+```python
+@strawberry.type
+class Query:
+    @strawberry.field
+    def name(self) -> str:
+        return "Strawberry"
+```
+
+## Arguments
+
+GraphQL fields can accept arguments, usually to filter out or retrieve specific
+objects:
+
+```python
+FRUITS = [
+    "Strawberry",
+    "Apple",
+    "Orange",
+]
+
+
+@strawberry.type
+class Query:
+    @strawberry.field
+    def fruit(self, startswith: str) -> str | None:
+        for fruit in FRUITS:
+            if fruit.startswith(startswith):
+                return fruit
+        return None
+```
+
+Additional metadata can be added to arguments like a different name and
```

Comment:
Just a suggestion on how I would have worded it. Feel free to ignore it :)

```suggestion
Additional metadata can be added to arguments, for example a custom name and
```

Reply:
I like it more